.github/workflows/commitint.yml

```diff
@@ -0,0 +1,23 @@
+name: commitlint
+
+on:
+  pull_request:
+
+jobs:
+  run-commitlint-on-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
as is done in ci.yml, we should be able to use `npm ci` here instead of `npm install`. `npm ci` is intended for automated environments because it doesn't try to update dependencies, only install them as exactly appears in package-lock.json https://stackoverflow.com/a/53325242